freeipa.spec.in

```diff
@@ -1323,6 +1331,45 @@ fi
 # with_selinux
 %endif
 
+%triggerin client -- sssd-common < 2.10
+# Has the client been configured?
+restore=0
```

Comment:
This variable is = 0 if client packages are installed but the host is not enrolled and =2 if the host is enrolled. It should be used as a condition to execute the replacement (if the client is not installed, skip). Otherwise the upgrade of freeipa-client package triggers the following error message:
```
  Downgrading      : sssd-common-2.10.0-0.240501.133809.gitd42c5e7da.fc39.x86_64                                                                                           10/42 
  Running scriptlet: sssd-common-2.10.0-0.240501.133809.gitd42c5e7da.fc39.x86_64                                                                                           10/42 
sed: can't read /etc/ssh/ssh_config.d/04-ipa.conf: No such file or directory
sed: can't read /etc/ssh/ssh_config.d/04-ipa.conf: No such file or directory
warning: %triggerin(freeipa-client-4.12.0.dev-0.fc39.x86_64) scriptlet failed, exit status 2

Error in <unknown> scriptlet in rpm package sssd-common

```

Reply:
thanks @flo-renaud , fixed.